build.psm1

```diff
@@ -2210,7 +2210,7 @@ function Start-CrossGen {
     $crossGenRequiredAssemblies = @("mscorlib.dll", "System.Private.CoreLib.dll")
```

Comment:
@SteveL-MSFT, your last commit had 1 failures in `PowerShell-CI-windows`
Verify approved aliases list.All present Cmdlets should have the correct ConfirmImpact
```powershell
Expected $null or empty, but got @(@{Name=Out-GridView; ConfirmImpact=; SideIndicator==>}, @{Name=Tee-Object; ConfirmImpact=None; SideIndicator=<=}).
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Basic\DefaultCommands.Tests.ps1: line 606
606:         $result | Should -BeNullOrEmpty
```